docs/reference/schema.md

```diff
@@ -93,6 +86,36 @@ Specifies the list of ingresses.
 
 Specifies the list of services. Applications must have at least one service.
 
+
+## Registry
+
+Allows storing the name of the container registry in configuration. This is used when building and deploying images for two purposes:
+
+- Determining how to tag the images
+- Determining where to push the images
+
+If this is not specified in configuration, interactive deployments will prompt for it.
+
+### Registry Example
+
+```yaml
+registry:
+  hostname: example.azurecr.io
```

Comment:
Maybe `name` is better here instead of `hostname`.

Reply:
`ConfigRegistry` and `ContainerRegistry` have a `Hostname` property. Shall I rename it to `Name` too?

Reply:
@jkotalik I assume you missed my question 😐

Reply:
If those are both hostname, I'm fine with keeping this as host name for now then.

Reply:
Ok, than I guess I have addresses all your change requests.